py/picca/test/test_helpers.py

```diff
@@ -0,0 +1,201 @@
+import configparser as ConfigParser
+from picca.utils import userprint
+import fitsio
+import os
+import numpy as np
+import h5py
+import sys
+
+
+def update_system_status_values(path, section, system, value):
+
+    ### Make ConfigParser case sensitive
+    class CaseConfigParser(ConfigParser.ConfigParser):
```

Comment:
why are you defining the class inside this function? Would it not be better to define it at the module level? Also docstring missing

Reply:
I didn't invent this, it's been inside the test case since forever. But indeed it works as well when doing it at the module level and looks cleaner

Reply:
hahaha sorry

Reply:
at least I now know what this is for and there's some mini-documentation